components/src/styles.css

```diff
@@ -0,0 +1,3 @@
+/* You can add global styles to this file, and also import other style files */
+
+@import "../../../src/scss/tailwind.css";
```

Comment:
This means that this package will only work in web for now. I think that's an acceptable limitation vs having to maintain two styles + tailwind configs.